internal/config/defaults.go

```diff
@@ -41,11 +41,11 @@ const (
 	DefGRPCKeepAlivePermitWithoutStream = true
 
 	// Client Backoff defaults
-	DefBackoffInitialInterval     = 500 * time.Millisecond
+	DefBackoffInitialInterval     = 10 * time.Second
 	DefBackoffRandomizationFactor = 0.5 // the value is 0 <= and < 1
 	DefBackoffMultiplier          = 1.5
```

Comment:
```suggestion
	DefBackoffInitialInterval     = 1 * time.Second
	DefBackoffRandomizationFactor = 0.5 // the value is 0 <= and < 1
	DefBackoffMultiplier          = 3
```

I think these setting may be better so that we retry quickly at the start and then the retry time grows larger over time